pages/annotation.vue

```diff
@@ -1,68 +1,58 @@
 <template>
 
-    <b-container fluid>
-
-        <!-- Navigation bar -->
-		<tool-navbar 
-			:navItems="pageData"
-			:navOrder="pageOrder"
-			:pageName="pageData.annotation.fullName">
-		</tool-navbar>
-
-        <b-row>
-            <b-col cols="4"></b-col>
-            <b-col cols="4">
-                <img class="vertical-center-mockup"
-                     :src="require(`~/assets/${mockupImage.filename}`)"
-                     :width="mockupImage.width"
-                     :height="mockupImage.height"/>
-            </b-col>
-            <b-col cols="4"></b-col>
-        </b-row>        
-
-    </b-container>
+  <b-container fluid>
+
+    <!-- Navigation bar -->
+    <tool-navbar
+      :navItems="pageData"
+      :navOrder="pageOrder"
+      :pageName="pageData.annotation.fullName">
+    </tool-navbar>
+
+    <!-- This gives us built-in keyboard navigation! -->
+    <b-tabs pills card vertical>
+      <!--      TODO: hardcode the pages and just toggle visibility based on state-->
+      <b-tab v-for="page in pages" :title="page.title" :key="page.id">
+        <b-card-text>
+          <component
+            :is="page.component"
+          ></component>
+        </b-card-text>
+      </b-tab>
+    </b-tabs>
+  </b-container>
 
 </template>
 
 <script>
-
-	// Allows for reference to store data by creating simple, implicit getters
-	import { mapState } from "vuex";
-
-    export default {
-
-        data() {
-
-            return {
-
-                // Image for temporary annotation page mockup
-                mockupImage: {
-                    filename: "annotation_mockup.jpg",
-                    height: 956,
-                    width: 1045
-                }
-            }
-        },
-
-        computed: {
-
-            ...mapState([
-
-                "pageData",
-                "pageOrder"
-            ])
-        }
-        
+// Allows for reference to store data by creating simple, implicit getters
+// Fields listed in mapState below can be found in the store (index.js)
+import { mapState } from "vuex";
+
+export default {
+  name: "Annotation",
+  data() {
+    return {
+      // TODO: "pages" is used as static testing data. Should be replaced with actual list of used categories
+      // from global state / store
+      pages: [
```

Comment:
We can commit this for now but should we be adding this to the store this week after these PRs?

Reply:
Up to you. I thought getting them in as is for now might be easier. But if you want we can also hook them up directly in this PR. The benefit of handling this in a separate PR might be that we won't have as many merge conflicts for the downstream PRs.

Reply:
Separate PR it is.